docs/json/sonarr/cf/anime-bd-tier-06-fansubs.json

```diff
@@ -66,7 +66,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(ANE)\\b"
+        "value": "\\[ANE\\]"
```

Comment:
would it make sense to use `-ANE$` as well?

Reply:
Probably yes